src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/MagicalZombiePills.java

```diff
@@ -74,4 +78,19 @@ public EntityInteractHandler getItemHandler() {
     public ItemUseHandler onRightClick() {
         return PlayerRightClickEvent::cancel;
     }
-}
\ No newline at end of file
+
+    private void healZombieVillager(@Nonnull ZombieVillager zombieVillager, @Nonnull Player p) {
+        zombieVillager.setConversionTime(1);
+
+        if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_15)) {
+            zombieVillager.setConversionPlayer(p);
+        }
+    }
+
+    private void healZombifiedPiglin(@Nonnull PigZombie zombiePiglin) {
```

Comment:
Wait a second, I only just realized that you removed the 1.16 version check? O_O

Reply:
entity.getType will never be equal to a ZombiePiglin, because they dont exist in other versions

Reply:
I could add it back though, if thats preferred

Reply:
> entity.getType will never be equal to a ZombiePiglin, because they dont exist in other versions

Yes and cause an exception

Reply:
I dont actually think it will create an exception? But sure Ill add the check

Reply:
> I dont actually think it will create an exception?  (im probably wrong here)
> But sure Ill add the check

Of course it will. The server will try to access an EntityType that doesn't exist and boom. Exception.

Reply:
Oh I was assuming that it still uses the latest version of the api, and that it would just throw if anything out of that version was accessed. Good to know this!